crates/ruff_python_formatter/src/expression/expr_bytes_literal.rs

```diff
@@ -0,0 +1,55 @@
+use ruff_formatter::FormatRuleWithOptions;
+use ruff_python_ast::AnyNodeRef;
+use ruff_python_ast::ExprBytesLiteral;
+
+use crate::comments::SourceComment;
+use crate::expression::expr_string_literal::is_multiline_string;
+use crate::expression::parentheses::{NeedsParentheses, OptionalParentheses};
+use crate::expression::string::{AnyString, FormatString, StringLayout};
+use crate::prelude::*;
+
+#[derive(Default)]
+pub struct FormatExprBytesLiteral {
+    layout: StringLayout,
```

Comment:
Is this required for byte literals as well?

Reply:
I checked that byte strings are not accepted as docstrings.

CC @MichaReiser